techmatters/flex-plugins
# Implement permissions configuration for masking PII

Primary reviewer: @stephenhand 

## Description
- Implemented permissions configuration for viewing / masking PII and identifiers - `viewIdentifiers`. 
- `viewIdentifiers` allows permission to view/mask all the aselo components where user handler, ip address and contact details appear and twilio strings when a cal/chat comes in. 
- Added permission configuration to all helplines with all being set to 'everyone'
- Updated test suite for `getPermissionsForViewingIdentifiers` handler 

### Checklist

### Related Issues
Fixes # 1471

### Verification steps
- Change permission config condition set for `dev` and check whether identifiers are masked.
